src/redux/users/usersSlice.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable */
+// import { createSlice } from "@reduxjs/toolkit";
+// import { createAsyncThunk } from "@reduxjs/toolkit";
+
+// const initialState = {
+//     loading: false,
+//     users: [],
+//     error: ''
+// }
+
+// const UserSlice = createSlice({
+//     name: 'user',
+//     initialState,
+// })
```

Comment:
- [ ] As previously suggested, please remove unused or commented pieces of code. It clutters your codebase and is a nightmare for anyone reading your code 👍